ext/RMagick/extconf.rb

```diff
@@ -186,6 +159,32 @@ def exit_failure(msg)
       exit(1)
     end
 
+    def determine_imagemagick_package
+      unless find_executable('pkg-config')
+        exit_failure "Can't install RMagick #{RMAGICK_VERS}. Can't find pkg-config in #{ENV['PATH']}\n"
+      end
+
+      packages = `pkg-config --list-all`.scan(/(ImageMagick\-6[\.A-Z0-9]+) .*/).flatten
+
+      # For ancient version of ImageMagick 6 we need a different regex
```

Comment:
@dlemstra which versions is this a problem for? Can you maybe add a version number to the comment so we know when we can remove this check? (FWIW, we currently only support back to 6.7)

Reply:
I needed to add this because support for ImageMagick was moved down to 6.7. I really don't know when this was changed. If I had to guess it would be around the time ImageMagick 7 was started.

Reply:
Okay, so we'll assume we need to keep this until we drop support for all versions of ImageMagick 6.

Reply:
That is correct.